torch/testing/_internal/distributed/rpc/rpc_test.py

```diff
@@ -4736,6 +4762,33 @@ def test_rpc_script_timeout(self):
 
 class TensorPipeAgentRpcTest(RpcAgentTestFixture):
 
+    @skip_if_lt_x_gpu(2)
```

Comment:
The test below doesn't appear to use GPUs, why is this decorator needed?

Reply:
I will remove.